src/moaicore/MOAIFreeTypeFont.cpp

```diff
@@ -343,10 +343,10 @@ int MOAIFreeTypeFont::_setDefaultSize(lua_State *L){
 FT_Face MOAIFreeTypeFont::AffirmFreeTypeFace () {
 	
 	if (!this->mFreeTypeFace) {
-		FT_Library library;
-		FT_Error error = FT_Init_FreeType( &library );
+		this->mLibrary = new FT_Library();
+		FT_Error error = FT_Init_FreeType( this->mLibrary );
 		CHECK_ERROR(error);
-		return this->LoadFreeTypeFace( &library );
+		return this->LoadFreeTypeFace( this->mLibrary );
```

Comment:
Is there risk then that we're leaking memory by calling `AffirmFreeTypeFace()` several times, since it'll allocate one new library each time and we only delete `mLibrary` once?